files/12dSolutions/Georef_A_12d.ifc

```diff
@@ -0,0 +1,313 @@
+#17 = IFCDIRECTION((0., 0., 1.));
+#18 = IFCDIRECTION((1., 0., 0.));
+#19 = IFCDIRECTION((0., 1.));
+#20 = IFCPROJECTEDCRS('MGA2020 Zone 55 with AHD', 'HD: MGA2020, Zone 55 VD: AHD', 'EPSG:7855', 'EPSG:5711', 'UTM', '55', #9);
+#21 = IFCMAPCONVERSIONSCALED(#13, #20, 16833194.0548, -4009644.0757, 0., 1., 0., 1., 1., 1., 1.);
+#22 = IFCRELDECLARES('1D9xN1LrTBnBhKKk9PYZe4', #2, '', '', #1, (#23, #35, #36));
+#23 = IFCACTOR('2AOiWG0JnFkxTQ1v1cSEhH', $, $, $, $, #4);
+#24 = IFCCLASSIFICATION('Uniclass 2015', '1', '23-04-2018', 'Uniclass', 'Unified construction classification', 'uniclass.thenbs.com', $);
+#25 = IFCRELASSOCIATESCLASSIFICATION('3Prl9hvE55tuuSwU13KxaX', #2, $, $, (#1), #24);
+#26 = IFCDOCUMENTINFORMATION('project_plan', 'project_plan.pdf', $, 'https://github.com/bSI-InfraRoom/IFC-Tunnel-Deployment/blob/main/project_plan.pdf', $, $, $, $, #27, (#29), $, $, 'application/pdf', $, $, $, $);
+#27 = IFCPERSON($, 'Mackellar', 'Adam', $, ('Mr.'), $, $, (#28));
+#28 = IFCADDRESS(.OFFICE., $, $);
+#29 = IFCORGANIZATION('bSI', 'Building Smart International', $, $, $);
+#30 = IFCRELASSOCIATESDOCUMENT('2cHYDqxHnFSPFq9VpDKy3d', #2, '', '', (#1), #26);
+#31 = IFCDATASETINFORMATION('Dataset-test', 'Dataset-test-xml', $, 'http://xxx.yyy.se/Datasets/Dataset-test.xml', $, $, $, $, $, $, $, $, 'text/xml', $, $, $, $, 'http://geosciml.org/geosciml/2.0/xsd/geosciml.xsd');
+#32 = IFCRELASSOCIATESDATASET('1lqFAJx4j29QdsRqCeOiV0', #2, '', '', (#1), #31);
+#33 = IFCLIBRARYINFORMATION('Example Library', '1.2.3', #3, '2023-03-10T13:05:38', 'http://xxx.yyy.se/Libraries/ExampleLibrary.ifc', $);
+#34 = IFCRELASSOCIATESLIBRARY('3XmwJzQb95L8UPunq1W38L', #2, '', '', (#1), #33);
+#35 = IFCARCHELEMENTTYPE('3l9$2hAxr6Hx$RitoS1PAI', #2, 'ArchType Example', $, $, $, $, $, 'MyArch', .USERDEFINED.);
+#36 = IFCPROPERTYSETTEMPLATE('30$TWUmhXCpQik7$iwa1Mi', #2, 'Pset_GeoObsBoreholeConstruction', $, .PSET_TYPEDRIVENOVERRIDE., 'IfcBorehole', (#37));
+#37 = IFCSIMPLEPROPERTYTEMPLATE('1lcv35HjrE1R$ee0lT6V6c', #2, 'DrillingDiameter', $, .P_SINGLEVALUE., 'IfcPositiveLengthMeasure', $, $, $, $, $, $);
+#38 = IFCRELAGGREGATES('3niR2H_xX4avsspSUfuF$r', #2, 'ProjectContainer', 'ProjectContainer for Sites', #1, (#39, #49));
+#39 = IFCTUNNEL('0Jd9SK1MLFwOa3VO4HUs_G', #2, 'Tunnel 01', $, $, #14, $, $, .ELEMENT., $);
+#40 = IFCRELAGGREGATES('37up2TQXD3WvrtWAE4PQU$', #2, 'Tunnel 01 Spatial Container', 'Tunnel 01 Container for Spatial Elements', #39, (#41));
+#41 = IFCTUNNELPART('04cmyVtbL7QOVIuSxhRxbG', #2, 'Tunnel Part 01', $, $, #14, $, $, .ELEMENT., $, $);
+#42 = IFCRELCONTAINEDINSPATIALSTRUCTURE('1EVwZ9jxnC8BIWMPnI$kP2', #2, 'Tunnel Part 01 Container', 'Tunnel Part 01 Container for Elements', (#43), #41);
+#43 = IFCBUILDINGELEMENTPROXY('2wTV1IXaf4Ev6UbKzFLN7o', #2, 'Super', $, 'Super', #14, #44, $, .NOTDEFINED.);
+#44 = IFCPRODUCTDEFINITIONSHAPE($, $, (#45));
+#45 = IFCSHAPEREPRESENTATION(#13, 'Body', 'Point', (#46));
+#46 = IFCCARTESIANPOINT((-16330482.0548, 10971354.0757, 0.));
```

Comment:
Looking at `#21 = IFCMAPCONVERSIONSCALED(#13, #20, 16833194.0548, -4009644.0757, 0., 1., 0., 1., 1., 1., 1.);` the coordinate values of the `IfcCartesianPoint` seem weird. Please double check with @LeeGregory12d.